docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>the C = I + s AA' will necessarily result in fairly "smooth" covariances, maybe it could be more interesting to have something a bit more peaked. Similarly, what is the dimension of A?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
(here by peak I mean getting rid of Identity altogether, and play with the size of A (i.e. a [Wishart distribution ](https://en.wikipedia.org/wiki/Wishart_distribution))

Reply:
ok